[2025-01-28T04:24:07.884+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-28T04:24:07.986+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: demo.airflow scheduled__2022-01-29T00:00:00+00:00 [queued]>
[2025-01-28T04:24:08.034+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: demo.airflow scheduled__2022-01-29T00:00:00+00:00 [queued]>
[2025-01-28T04:24:08.039+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-28T04:24:08.912+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): airflow> on 2022-01-29 00:00:00+00:00
[2025-01-28T04:24:08.992+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'demo', '***', 'scheduled__2022-01-29T00:00:00+00:00', '--job-id', '76', '--raw', '--subdir', 'DAGS_FOLDER/documentation_dag.py', '--cfg-path', '/tmp/tmpnhnft2b4']
[2025-01-28T04:24:08.998+0000] {standard_task_runner.py:105} INFO - Job 76: Subtask ***
[2025-01-28T04:24:08.998+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=236) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-28T04:24:09.001+0000] {standard_task_runner.py:72} INFO - Started process 273 to run task
[2025-01-28T04:24:09.183+0000] {task_command.py:467} INFO - Running <TaskInstance: demo.airflow scheduled__2022-01-29T00:00:00+00:00 [running]> on host a845ebe66097
[2025-01-28T04:24:09.566+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='demo' AIRFLOW_CTX_TASK_ID='***' AIRFLOW_CTX_EXECUTION_DATE='2022-01-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-01-29T00:00:00+00:00'
[2025-01-28T04:24:09.567+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-28T04:24:09.568+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-28T04:24:09.568+0000] {logging_mixin.py:190} INFO - Current task name:*** state:running start_date:2025-01-28 04:24:07.987004+00:00
[2025-01-28T04:24:09.569+0000] {logging_mixin.py:190} INFO - Dag name:demo and current dag run status:running
[2025-01-28T04:24:09.575+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-28T04:24:09.576+0000] {logging_mixin.py:190} INFO - ***
[2025-01-28T04:24:09.577+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-28T04:24:09.622+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-28T04:24:09.623+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=demo, task_id=***, run_id=scheduled__2022-01-29T00:00:00+00:00, execution_date=20220129T000000, start_date=20250128T042407, end_date=20250128T042409
[2025-01-28T04:24:09.649+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-28T04:24:09.649+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-28T04:24:09.650+0000] {logging_mixin.py:190} INFO - Dag name:demo queued_at:2025-01-28 04:23:52.326770+00:00
[2025-01-28T04:24:09.650+0000] {logging_mixin.py:190} INFO - Task hostname:a845ebe66097 operator:_PythonDecoratedOperator
[2025-01-28T04:24:09.729+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-28T04:24:09.786+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-28T04:24:09.796+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
